src/azure/Templates/AzureCsprojTemplate.cshtml

```diff
@@ -0,0 +1,73 @@
+﻿@using System
+@using System.Linq
+@using AutoRest.Core.Utilities
+@using AutoRest.CSharp.Azure.Model
+@using AutoRest.CSharp.Model
+@inherits AutoRest.Core.Template<AutoRest.CSharp.Azure.Model.CodeModelCsa>
+
+@{
+    var serviceName = Model.ServiceName;
+}
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <PackageId>Microsoft.Azure.Management.@(serviceName).Fluent</PackageId>
+        <Version>@Model.ClientPackageVersion</Version>
+        <AssemblyName>Microsoft.Azure.Management.@(serviceName).Fluent</AssemblyName>
+        <RootNamespace>Microsoft.Azure.Management.@(serviceName).Fluent</RootNamespace>
+        <AssemblyTitle>Microsoft Azure @(serviceName) Management Library</AssemblyTitle>
+        <Authors>Microsoft</Authors>
+        <Description>Provides @(serviceName) service management (Fluent) capabilities for Microsoft Azure.</Description>
+        <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</PackageIconUrl>
+        <PackageTags>Microsoft Azure @(serviceName) management;@(serviceName.ToLower());REST HTTP client;azureofficial;windowsazureofficial;Fluent</PackageTags>
+        <PackageProjectUrl>https://github.com/Azure/azure-sdk-for-net/tree/Fluent</PackageProjectUrl>
+        <PackageLicenseUrl>https://raw.githubusercontent.com/Microsoft/dotnet/master/LICENSE</PackageLicenseUrl>
+        <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+        <GenerateDocumentationFile>false</GenerateDocumentationFile>
+        <SignAssembly>true</SignAssembly>
+        <DelaySign>true</DelaySign>
+        <AssemblyOriginatorKeyFile>../../../tools/MSSharedLibKey.snk</AssemblyOriginatorKeyFile>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.3' ">$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+        <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.3' ">1.6.1</NetStandardImplicitPackageVersion>
+        <TargetFrameworks>net452;netstandard1.3</TargetFrameworks>
+    </PropertyGroup>
+
+    @* The "Version=" needs to be inside a C# expression in order to prevent the compiled template from calling WriteAttributeValue which causes a build error. *@
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" @("Version=\"" + @Model.ClientRuntimeVersion + "\"") />
+        <PackageReference Include="Microsoft.Azure.Management.ResourceManager.Fluent" @("Version=\"" + @Model.ClientPackageVersion + "\"") />
```

Comment:
this template contains fluent-specific stuff so it should go into the `azurefluent` folder ;)